src/Run/Setup.cs

```diff
@@ -111,7 +153,7 @@ private string BuildParametersForCommand(Dictionary<string, string> commandParam
             {
                 if (!parameters.Key.Equals("toolName") && !string.IsNullOrEmpty(parameters.Value))
                 {
-                    string value = parameters.Value.Equals("default") ? FindSettingValue(parameters.Key) : parameters.Value;
+                    string value = parameters.Value.Equals("default") ? FindSettingValue(parameters.Key) : ParseSettingValue(parameters.Value);
```

Comment:
Discussed offline with @maririos, and we decided to not impose rules on where the ${} can be used, but leave it to the config.json author.